data/mods/Tolerate_This/cooking_components.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "batter",
+    "type": "requirement",
+    "//": "Bread type food usable as batter for frying.",
+    "components": [
+      [
+        [ "flour", 1 ],
+        [ "cornmeal", 1 ],
+		[ "rice_flour", 1 ],
```

Comment:
Remove the tabs here and replace with spaces

Reply:
Thanks for that catch... I thought my editor would be text spacing only, not tabs. It is also being really helpful and auto indenting... which is great, except it's adding tabs. XD

Should be fixed and should also show my whitespace/tabs now just in case now for me to check when editing.